libavformat/hls.c

```diff
@@ -843,6 +846,27 @@ static int parse_playlist(HLSContext *c, const char *url,
             ptr = strchr(ptr, '@');
             if (ptr)
                 seg_offset = strtoll(ptr+1, NULL, 10);
+        } else if (av_strstart(line, "#EXT-X-PROGRAM-DATE-TIME:", &ptr)) {
+            struct tm pdt;
+            int y,M,d,h,m,s;
+            double ms;
+
+            // TODO: take timezone into consideration
```

Comment:
Oh. Got it. I was thinking about that. So is the assumption at the moment, that it is always UTC? What timezone does the UK example docker container use?

Reply:
They were using UTC as well.

```
#EXT-X-PROGRAM-DATE-TIME:2021-02-09T01:17:31.100+0000
```

We're basically ignoring `+0000` at the moment.